Aptoide/aptoide-client-v8
# [APP-1310] flavour cobrand

**What does this PR do?**

This PR aims at creating the cobrand flavour as well as making some configurations.
Changes package name, version name, theme (only putting background with the blue color as the design is not defined yet). 

**Database changed?**

 No

**Where should the reviewer start?**

- [ ] BUILD.GRADLE.KTS (app)

**How should this be manually tested?**

  Run the app in the different flavours and test it. Notice that at the moment, there is a question regarding passing the theme between the different modules, as they don't depend on each other so we can not share buildconfig files.
I alreayd have a workaround for this, but it needs to be discussed.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-1310](https://aptoide.atlassian.net/browse/APP-1310)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
